### DISCUSSION TITLE
Disable Individual Barrel File Flag w/ useMultipleFiles

### DISCUSSION THREAD

**Discussion by 0xcadams** [chrishoermann/zod-prisma-types#148] [Ideas]
@chrishoermann thanks for all of your hard work on this project. One idea for the `useMultipleFiles` flag - is there any way this could support *not* creating the barrel file in the index of each split output (`inputTypeSchemas/index.ts`, `modelSchema/index.ts`, etc). It's a quality of life improvement, but not a huge deal - the only reason is that we have a huge Prisma file, and the IDE loves to try to use the barrel file imports versus importing the split file directly. When we accidentally import one of those barrel files, the Typescript compiler slows to a crawl and the IDE basically hangs.

**Comment by chrishoermann** [comment]
@0xcadams Thanks for the suggestion. This should work now in the latest version as stated in the [docs](https://github.com/chrishoermann/zod-prisma-types#writebarrelfiles). hope this helps with your DX issue 😉

**Reply by 0xcadams** [reply]
Thanks! Can't believe I missed this 🙃

**Reply by chrishoermann** [reply]
You didn't miss it. I just added this option yesterday 😃

**Reply by 0xcadams** [reply]
Thanks for the quick addition!! 🤯
